From 5724a4ad7a27f7af31aa725694d3df09a68bb213 Mon Sep 17 00:00:00 2001
From: Jenkins <dev-kitchen@cloudera.com>
Date: Tue, 13 Mar 2018 18:03:08 -0700
Subject: [PATCH 2748/2748] Branching for 5.14.2 on Tue Mar 13 18:02:22 PDT
 2018

JOB_NAME : 'Cut-Release-Branches'
BUILD_NUMBER : '528'
CODE_BRANCH : ''
OLD_CDH_BRANCH : 'cdh5_5.14.x'

Pushed to remote common	git@github.sf.cloudera.com:Mirrors/apache-hadoop-common.git (push)
origin	git@github.sf.cloudera.com:CDH/hadoop.git (push)
---
 hadoop-assemblies/pom.xml                          |    4 ++--
 hadoop-build-tools/pom.xml                         |    2 +-
 hadoop-client/pom.xml                              |    4 ++--
 hadoop-common-project/hadoop-annotations/pom.xml   |    4 ++--
 hadoop-common-project/hadoop-auth-examples/pom.xml |    4 ++--
 hadoop-common-project/hadoop-auth/pom.xml          |    4 ++--
 hadoop-common-project/hadoop-common/pom.xml        |    4 ++--
 hadoop-common-project/hadoop-kms/pom.xml           |    4 ++--
 hadoop-common-project/hadoop-minikdc/pom.xml       |    4 ++--
 hadoop-common-project/hadoop-nfs/pom.xml           |    4 ++--
 hadoop-common-project/pom.xml                      |    4 ++--
 hadoop-dist/pom.xml                                |    4 ++--
 hadoop-hdfs-project/hadoop-hdfs-httpfs/pom.xml     |    4 ++--
 hadoop-hdfs-project/hadoop-hdfs-nfs/pom.xml        |    4 ++--
 hadoop-hdfs-project/hadoop-hdfs/pom.xml            |    4 ++--
 .../hadoop-hdfs/src/contrib/bkjournal/pom.xml      |    4 ++--
 hadoop-hdfs-project/pom.xml                        |    4 ++--
 .../hadoop-mapreduce-client-app/pom.xml            |    4 ++--
 .../hadoop-mapreduce-client-common/pom.xml         |    4 ++--
 .../hadoop-mapreduce-client-core/pom.xml           |    4 ++--
 .../hadoop-mapreduce-client-hs-plugins/pom.xml     |    4 ++--
 .../hadoop-mapreduce-client-hs/pom.xml             |    4 ++--
 .../hadoop-mapreduce-client-jobclient/pom.xml      |    4 ++--
 .../hadoop-mapreduce-client-nativetask/pom.xml     |    4 ++--
 .../hadoop-mapreduce-client-shuffle/pom.xml        |    4 ++--
 .../hadoop-mapreduce-client/pom.xml                |    4 ++--
 .../hadoop-mapreduce-examples/pom.xml              |    4 ++--
 hadoop-mapreduce-project/pom.xml                   |    4 ++--
 hadoop-mapreduce1-project/build.xml                |    2 +-
 hadoop-mapreduce1-project/cloudera-pom.xml         |    4 ++--
 .../cloudera/maven-packaging/hadoop-client/pom.xml |    4 ++--
 .../cloudera/maven-packaging/hadoop-core/pom.xml   |    4 ++--
 .../maven-packaging/hadoop-examples/pom.xml        |    4 ++--
 .../maven-packaging/hadoop-minicluster/pom.xml     |    4 ++--
 .../cloudera/maven-packaging/hadoop-mrunit/pom.xml |    4 ++--
 .../maven-packaging/hadoop-streaming/pom.xml       |    4 ++--
 .../maven-packaging/hadoop-tarball/pom.xml         |    4 ++--
 .../cloudera/maven-packaging/hadoop-test/pom.xml   |    4 ++--
 .../cloudera/maven-packaging/hadoop-tools/pom.xml  |    4 ++--
 .../cloudera/maven-packaging/pom.xml               |    4 ++--
 hadoop-mapreduce1-project/ivy/libraries.properties |    4 ++--
 hadoop-maven-plugins/pom.xml                       |    2 +-
 hadoop-minicluster/pom.xml                         |    4 ++--
 hadoop-project-dist/pom.xml                        |    4 ++--
 hadoop-project/pom.xml                             |    4 ++--
 hadoop-tools/hadoop-ant/pom.xml                    |    4 ++--
 hadoop-tools/hadoop-archive-logs/pom.xml           |    4 ++--
 hadoop-tools/hadoop-archives/pom.xml               |    4 ++--
 hadoop-tools/hadoop-aws/pom.xml                    |    4 ++--
 hadoop-tools/hadoop-azure-datalake/pom.xml         |    2 +-
 hadoop-tools/hadoop-azure/pom.xml                  |    4 ++--
 hadoop-tools/hadoop-datajoin/pom.xml               |    4 ++--
 hadoop-tools/hadoop-distcp/pom.xml                 |    4 ++--
 hadoop-tools/hadoop-extras/pom.xml                 |    4 ++--
 hadoop-tools/hadoop-gridmix/pom.xml                |    4 ++--
 hadoop-tools/hadoop-openstack/pom.xml              |    4 ++--
 hadoop-tools/hadoop-pipes/pom.xml                  |    4 ++--
 hadoop-tools/hadoop-rumen/pom.xml                  |    4 ++--
 hadoop-tools/hadoop-sls/pom.xml                    |    4 ++--
 hadoop-tools/hadoop-streaming/pom.xml              |    4 ++--
 hadoop-tools/hadoop-tools-dist/pom.xml             |    4 ++--
 hadoop-tools/pom.xml                               |    4 ++--
 .../hadoop-yarn/hadoop-yarn-api/pom.xml            |    4 ++--
 .../pom.xml                                        |    4 ++--
 .../pom.xml                                        |    4 ++--
 .../hadoop-yarn/hadoop-yarn-applications/pom.xml   |    4 ++--
 .../hadoop-yarn/hadoop-yarn-client/pom.xml         |    4 ++--
 .../hadoop-yarn/hadoop-yarn-common/pom.xml         |    4 ++--
 .../hadoop-yarn/hadoop-yarn-registry/pom.xml       |    4 ++--
 .../pom.xml                                        |    4 ++--
 .../hadoop-yarn-server-common/pom.xml              |    4 ++--
 .../hadoop-yarn-server-nodemanager/pom.xml         |    4 ++--
 .../hadoop-yarn-server-resourcemanager/pom.xml     |    4 ++--
 .../hadoop-yarn-server-tests/pom.xml               |    4 ++--
 .../hadoop-yarn-server-web-proxy/pom.xml           |    4 ++--
 .../hadoop-yarn/hadoop-yarn-server/pom.xml         |    4 ++--
 .../hadoop-yarn/hadoop-yarn-site/pom.xml           |    4 ++--
 hadoop-yarn-project/hadoop-yarn/pom.xml            |    4 ++--
 hadoop-yarn-project/pom.xml                        |    4 ++--
 pom.xml                                            |    4 ++--
 80 files changed, 156 insertions(+), 156 deletions(-)

diff --git a/hadoop-assemblies/pom.xml b/hadoop-assemblies/pom.xml
index ad946f0..2dba253 100644
--- a/hadoop-assemblies/pom.xml
+++ b/hadoop-assemblies/pom.xml
@@ -23,12 +23,12 @@
   <parent>
     <groupId>org.apache.hadoop</groupId>
     <artifactId>hadoop-project</artifactId>
-    <version>2.6.0-cdh5.14.1-SNAPSHOT</version>
+    <version>2.6.0-cdh5.14.2</version>
     <relativePath>../hadoop-project</relativePath>
   </parent>
   <groupId>org.apache.hadoop</groupId>
   <artifactId>hadoop-assemblies</artifactId>
-  <version>2.6.0-cdh5.14.1-SNAPSHOT</version>
+  <version>2.6.0-cdh5.14.2</version>
   <name>Apache Hadoop Assemblies</name>
   <description>Apache Hadoop Assemblies</description>
 
diff --git a/hadoop-build-tools/pom.xml b/hadoop-build-tools/pom.xml
index 92d6e59..6de4bc7 100644
--- a/hadoop-build-tools/pom.xml
+++ b/hadoop-build-tools/pom.xml
@@ -18,7 +18,7 @@
   <parent>
     <artifactId>hadoop-main</artifactId>
     <groupId>org.apache.hadoop</groupId>
-    <version>2.6.0-cdh5.14.1-SNAPSHOT</version>
+    <version>2.6.0-cdh5.14.2</version>
   </parent>
   <modelVersion>4.0.0</modelVersion>
   <artifactId>hadoop-build-tools</artifactId>
diff --git a/hadoop-client/pom.xml b/hadoop-client/pom.xml
index 6a0b6c2..d7ffe3c 100644
--- a/hadoop-client/pom.xml
+++ b/hadoop-client/pom.xml
@@ -18,12 +18,12 @@
   <parent>
     <groupId>org.apache.hadoop</groupId>
     <artifactId>hadoop-project-dist</artifactId>
-    <version>2.6.0-cdh5.14.1-SNAPSHOT</version>
+    <version>2.6.0-cdh5.14.2</version>
     <relativePath>../hadoop-project-dist</relativePath>
   </parent>
   <groupId>org.apache.hadoop</groupId>
   <artifactId>hadoop-client</artifactId>
-  <version>2.6.0-cdh5.14.1-SNAPSHOT</version>
+  <version>2.6.0-cdh5.14.2</version>
   <packaging>jar</packaging>
 
   <description>Apache Hadoop Client</description>
diff --git a/hadoop-common-project/hadoop-annotations/pom.xml b/hadoop-common-project/hadoop-annotations/pom.xml
index 72b6a83..f926686 100644
--- a/hadoop-common-project/hadoop-annotations/pom.xml
+++ b/hadoop-common-project/hadoop-annotations/pom.xml
@@ -20,12 +20,12 @@
   <parent>
     <groupId>org.apache.hadoop</groupId>
     <artifactId>hadoop-project</artifactId>
-    <version>2.6.0-cdh5.14.1-SNAPSHOT</version>
+    <version>2.6.0-cdh5.14.2</version>
     <relativePath>../../hadoop-project</relativePath>
   </parent>
   <groupId>org.apache.hadoop</groupId>
   <artifactId>hadoop-annotations</artifactId>
-  <version>2.6.0-cdh5.14.1-SNAPSHOT</version>
+  <version>2.6.0-cdh5.14.2</version>
   <description>Apache Hadoop Annotations</description>
   <name>Apache Hadoop Annotations</name>
   <packaging>jar</packaging>
diff --git a/hadoop-common-project/hadoop-auth-examples/pom.xml b/hadoop-common-project/hadoop-auth-examples/pom.xml
index 244326f..7e2ab89 100644
--- a/hadoop-common-project/hadoop-auth-examples/pom.xml
+++ b/hadoop-common-project/hadoop-auth-examples/pom.xml
@@ -20,12 +20,12 @@
   <parent>
     <groupId>org.apache.hadoop</groupId>
     <artifactId>hadoop-project</artifactId>
-    <version>2.6.0-cdh5.14.1-SNAPSHOT</version>
+    <version>2.6.0-cdh5.14.2</version>
     <relativePath>../../hadoop-project</relativePath>
   </parent>
   <groupId>org.apache.hadoop</groupId>
   <artifactId>hadoop-auth-examples</artifactId>
-  <version>2.6.0-cdh5.14.1-SNAPSHOT</version>
+  <version>2.6.0-cdh5.14.2</version>
   <packaging>war</packaging>
 
   <name>Apache Hadoop Auth Examples</name>
diff --git a/hadoop-common-project/hadoop-auth/pom.xml b/hadoop-common-project/hadoop-auth/pom.xml
index 8347ae8..bb10032 100644
--- a/hadoop-common-project/hadoop-auth/pom.xml
+++ b/hadoop-common-project/hadoop-auth/pom.xml
@@ -20,12 +20,12 @@
   <parent>
     <groupId>org.apache.hadoop</groupId>
     <artifactId>hadoop-project</artifactId>
-    <version>2.6.0-cdh5.14.1-SNAPSHOT</version>
+    <version>2.6.0-cdh5.14.2</version>
     <relativePath>../../hadoop-project</relativePath>
   </parent>
   <groupId>org.apache.hadoop</groupId>
   <artifactId>hadoop-auth</artifactId>
-  <version>2.6.0-cdh5.14.1-SNAPSHOT</version>
+  <version>2.6.0-cdh5.14.2</version>
   <packaging>jar</packaging>
 
   <name>Apache Hadoop Auth</name>
diff --git a/hadoop-common-project/hadoop-common/pom.xml b/hadoop-common-project/hadoop-common/pom.xml
index 0006e00..0649d07 100644
--- a/hadoop-common-project/hadoop-common/pom.xml
+++ b/hadoop-common-project/hadoop-common/pom.xml
@@ -20,12 +20,12 @@
   <parent>
     <groupId>org.apache.hadoop</groupId>
     <artifactId>hadoop-project-dist</artifactId>
-    <version>2.6.0-cdh5.14.1-SNAPSHOT</version>
+    <version>2.6.0-cdh5.14.2</version>
     <relativePath>../../hadoop-project-dist</relativePath>
   </parent>
   <groupId>org.apache.hadoop</groupId>
   <artifactId>hadoop-common</artifactId>
-  <version>2.6.0-cdh5.14.1-SNAPSHOT</version>
+  <version>2.6.0-cdh5.14.2</version>
   <description>Apache Hadoop Common</description>
   <name>Apache Hadoop Common</name>
   <packaging>jar</packaging>
diff --git a/hadoop-common-project/hadoop-kms/pom.xml b/hadoop-common-project/hadoop-kms/pom.xml
index 2dc24fb..6c2020e 100644
--- a/hadoop-common-project/hadoop-kms/pom.xml
+++ b/hadoop-common-project/hadoop-kms/pom.xml
@@ -22,12 +22,12 @@
   <parent>
     <groupId>org.apache.hadoop</groupId>
     <artifactId>hadoop-project</artifactId>
-    <version>2.6.0-cdh5.14.1-SNAPSHOT</version>
+    <version>2.6.0-cdh5.14.2</version>
     <relativePath>../../hadoop-project</relativePath>
   </parent>
   <groupId>org.apache.hadoop</groupId>
   <artifactId>hadoop-kms</artifactId>
-  <version>2.6.0-cdh5.14.1-SNAPSHOT</version>
+  <version>2.6.0-cdh5.14.2</version>
   <packaging>war</packaging>
 
   <name>Apache Hadoop KMS</name>
diff --git a/hadoop-common-project/hadoop-minikdc/pom.xml b/hadoop-common-project/hadoop-minikdc/pom.xml
index fd412ba..057c60c 100644
--- a/hadoop-common-project/hadoop-minikdc/pom.xml
+++ b/hadoop-common-project/hadoop-minikdc/pom.xml
@@ -18,13 +18,13 @@
   <parent>
     <groupId>org.apache.hadoop</groupId>
     <artifactId>hadoop-project</artifactId>
-    <version>2.6.0-cdh5.14.1-SNAPSHOT</version>
+    <version>2.6.0-cdh5.14.2</version>
     <relativePath>../../hadoop-project</relativePath>
   </parent>
   <modelVersion>4.0.0</modelVersion>
   <groupId>org.apache.hadoop</groupId>
   <artifactId>hadoop-minikdc</artifactId>
-  <version>2.6.0-cdh5.14.1-SNAPSHOT</version>
+  <version>2.6.0-cdh5.14.2</version>
   <description>Apache Hadoop MiniKDC</description>
   <name>Apache Hadoop MiniKDC</name>
   <packaging>jar</packaging>
diff --git a/hadoop-common-project/hadoop-nfs/pom.xml b/hadoop-common-project/hadoop-nfs/pom.xml
index 14ecaef..36725ed 100644
--- a/hadoop-common-project/hadoop-nfs/pom.xml
+++ b/hadoop-common-project/hadoop-nfs/pom.xml
@@ -20,12 +20,12 @@
   <parent>
     <groupId>org.apache.hadoop</groupId>
     <artifactId>hadoop-project</artifactId>
-    <version>2.6.0-cdh5.14.1-SNAPSHOT</version>
+    <version>2.6.0-cdh5.14.2</version>
     <relativePath>../../hadoop-project</relativePath>
   </parent>
   <groupId>org.apache.hadoop</groupId>
   <artifactId>hadoop-nfs</artifactId>
-  <version>2.6.0-cdh5.14.1-SNAPSHOT</version>
+  <version>2.6.0-cdh5.14.2</version>
   <packaging>jar</packaging>
 
   <name>Apache Hadoop NFS</name>
diff --git a/hadoop-common-project/pom.xml b/hadoop-common-project/pom.xml
index 1a874c2..2a1ab5a 100644
--- a/hadoop-common-project/pom.xml
+++ b/hadoop-common-project/pom.xml
@@ -20,12 +20,12 @@
   <parent>
     <groupId>org.apache.hadoop</groupId>
     <artifactId>hadoop-project</artifactId>
-    <version>2.6.0-cdh5.14.1-SNAPSHOT</version>
+    <version>2.6.0-cdh5.14.2</version>
     <relativePath>../hadoop-project</relativePath>
   </parent>
   <groupId>org.apache.hadoop</groupId>
   <artifactId>hadoop-common-project</artifactId>
-  <version>2.6.0-cdh5.14.1-SNAPSHOT</version>
+  <version>2.6.0-cdh5.14.2</version>
   <description>Apache Hadoop Common Project</description>
   <name>Apache Hadoop Common Project</name>
   <packaging>pom</packaging>
diff --git a/hadoop-dist/pom.xml b/hadoop-dist/pom.xml
index d486fa8..fe208c8 100644
--- a/hadoop-dist/pom.xml
+++ b/hadoop-dist/pom.xml
@@ -20,12 +20,12 @@
   <parent>
     <groupId>org.apache.hadoop</groupId>
     <artifactId>hadoop-project</artifactId>
-    <version>2.6.0-cdh5.14.1-SNAPSHOT</version>
+    <version>2.6.0-cdh5.14.2</version>
     <relativePath>../hadoop-project</relativePath>
   </parent>
   <groupId>org.apache.hadoop</groupId>
   <artifactId>hadoop-dist</artifactId>
-  <version>2.6.0-cdh5.14.1-SNAPSHOT</version>
+  <version>2.6.0-cdh5.14.2</version>
   <description>Apache Hadoop Distribution</description>
   <name>Apache Hadoop Distribution</name>
   <packaging>jar</packaging>
diff --git a/hadoop-hdfs-project/hadoop-hdfs-httpfs/pom.xml b/hadoop-hdfs-project/hadoop-hdfs-httpfs/pom.xml
index 9a61f08..30c5bf8 100644
--- a/hadoop-hdfs-project/hadoop-hdfs-httpfs/pom.xml
+++ b/hadoop-hdfs-project/hadoop-hdfs-httpfs/pom.xml
@@ -22,12 +22,12 @@
   <parent>
     <groupId>org.apache.hadoop</groupId>
     <artifactId>hadoop-project</artifactId>
-    <version>2.6.0-cdh5.14.1-SNAPSHOT</version>
+    <version>2.6.0-cdh5.14.2</version>
     <relativePath>../../hadoop-project</relativePath>
   </parent>
   <groupId>org.apache.hadoop</groupId>
   <artifactId>hadoop-hdfs-httpfs</artifactId>
-  <version>2.6.0-cdh5.14.1-SNAPSHOT</version>
+  <version>2.6.0-cdh5.14.2</version>
   <packaging>war</packaging>
 
   <name>Apache Hadoop HttpFS</name>
diff --git a/hadoop-hdfs-project/hadoop-hdfs-nfs/pom.xml b/hadoop-hdfs-project/hadoop-hdfs-nfs/pom.xml
index 7c1a327..3b4c9d5 100644
--- a/hadoop-hdfs-project/hadoop-hdfs-nfs/pom.xml
+++ b/hadoop-hdfs-project/hadoop-hdfs-nfs/pom.xml
@@ -20,12 +20,12 @@ http://maven.apache.org/xsd/maven-4.0.0.xsd">
   <parent>
     <groupId>org.apache.hadoop</groupId>
     <artifactId>hadoop-project</artifactId>
-    <version>2.6.0-cdh5.14.1-SNAPSHOT</version>
+    <version>2.6.0-cdh5.14.2</version>
     <relativePath>../../hadoop-project</relativePath>
   </parent>
   <groupId>org.apache.hadoop</groupId>
   <artifactId>hadoop-hdfs-nfs</artifactId>
-  <version>2.6.0-cdh5.14.1-SNAPSHOT</version>
+  <version>2.6.0-cdh5.14.2</version>
   <description>Apache Hadoop HDFS-NFS</description>
   <name>Apache Hadoop HDFS-NFS</name>
   <packaging>jar</packaging>
diff --git a/hadoop-hdfs-project/hadoop-hdfs/pom.xml b/hadoop-hdfs-project/hadoop-hdfs/pom.xml
index 8953a8e..b470e96 100644
--- a/hadoop-hdfs-project/hadoop-hdfs/pom.xml
+++ b/hadoop-hdfs-project/hadoop-hdfs/pom.xml
@@ -20,12 +20,12 @@ http://maven.apache.org/xsd/maven-4.0.0.xsd">
   <parent>
     <groupId>org.apache.hadoop</groupId>
     <artifactId>hadoop-project-dist</artifactId>
-    <version>2.6.0-cdh5.14.1-SNAPSHOT</version>
+    <version>2.6.0-cdh5.14.2</version>
     <relativePath>../../hadoop-project-dist</relativePath>
   </parent>
   <groupId>org.apache.hadoop</groupId>
   <artifactId>hadoop-hdfs</artifactId>
-  <version>2.6.0-cdh5.14.1-SNAPSHOT</version>
+  <version>2.6.0-cdh5.14.2</version>
   <description>Apache Hadoop HDFS</description>
   <name>Apache Hadoop HDFS</name>
   <packaging>jar</packaging>
diff --git a/hadoop-hdfs-project/hadoop-hdfs/src/contrib/bkjournal/pom.xml b/hadoop-hdfs-project/hadoop-hdfs/src/contrib/bkjournal/pom.xml
index 55db504..e6f58c5 100644
--- a/hadoop-hdfs-project/hadoop-hdfs/src/contrib/bkjournal/pom.xml
+++ b/hadoop-hdfs-project/hadoop-hdfs/src/contrib/bkjournal/pom.xml
@@ -20,13 +20,13 @@ http://maven.apache.org/xsd/maven-4.0.0.xsd">
   <parent>
     <groupId>org.apache.hadoop</groupId>
     <artifactId>hadoop-project</artifactId>
-    <version>2.6.0-cdh5.14.1-SNAPSHOT</version>
+    <version>2.6.0-cdh5.14.2</version>
     <relativePath>../../../../../hadoop-project</relativePath>
   </parent>
 
   <groupId>org.apache.hadoop.contrib</groupId>
   <artifactId>hadoop-hdfs-bkjournal</artifactId>
-  <version>2.6.0-cdh5.14.1-SNAPSHOT</version>
+  <version>2.6.0-cdh5.14.2</version>
   <description>Apache Hadoop HDFS BookKeeper Journal</description>
   <name>Apache Hadoop HDFS BookKeeper Journal</name>
   <packaging>jar</packaging>
diff --git a/hadoop-hdfs-project/pom.xml b/hadoop-hdfs-project/pom.xml
index a9d9113..bd71ca6 100644
--- a/hadoop-hdfs-project/pom.xml
+++ b/hadoop-hdfs-project/pom.xml
@@ -20,12 +20,12 @@ http://maven.apache.org/xsd/maven-4.0.0.xsd">
   <parent>
     <groupId>org.apache.hadoop</groupId>
     <artifactId>hadoop-project</artifactId>
-    <version>2.6.0-cdh5.14.1-SNAPSHOT</version>
+    <version>2.6.0-cdh5.14.2</version>
     <relativePath>../hadoop-project</relativePath>
   </parent>
   <groupId>org.apache.hadoop</groupId>
   <artifactId>hadoop-hdfs-project</artifactId>
-  <version>2.6.0-cdh5.14.1-SNAPSHOT</version>
+  <version>2.6.0-cdh5.14.2</version>
   <description>Apache Hadoop HDFS Project</description>
   <name>Apache Hadoop HDFS Project</name>
   <packaging>pom</packaging>
diff --git a/hadoop-mapreduce-project/hadoop-mapreduce-client/hadoop-mapreduce-client-app/pom.xml b/hadoop-mapreduce-project/hadoop-mapreduce-client/hadoop-mapreduce-client-app/pom.xml
index a5261ae..3ae1072 100644
--- a/hadoop-mapreduce-project/hadoop-mapreduce-client/hadoop-mapreduce-client-app/pom.xml
+++ b/hadoop-mapreduce-project/hadoop-mapreduce-client/hadoop-mapreduce-client-app/pom.xml
@@ -19,12 +19,12 @@
   <parent>
     <artifactId>hadoop-mapreduce-client</artifactId>
     <groupId>org.apache.hadoop</groupId>
-    <version>2.6.0-cdh5.14.1-SNAPSHOT</version>
+    <version>2.6.0-cdh5.14.2</version>
   </parent>
   <modelVersion>4.0.0</modelVersion>
   <groupId>org.apache.hadoop</groupId>
   <artifactId>hadoop-mapreduce-client-app</artifactId>
-  <version>2.6.0-cdh5.14.1-SNAPSHOT</version>
+  <version>2.6.0-cdh5.14.2</version>
   <name>hadoop-mapreduce-client-app</name>
 
   <properties>
diff --git a/hadoop-mapreduce-project/hadoop-mapreduce-client/hadoop-mapreduce-client-common/pom.xml b/hadoop-mapreduce-project/hadoop-mapreduce-client/hadoop-mapreduce-client-common/pom.xml
index 328cb47..531a0b0 100644
--- a/hadoop-mapreduce-project/hadoop-mapreduce-client/hadoop-mapreduce-client-common/pom.xml
+++ b/hadoop-mapreduce-project/hadoop-mapreduce-client/hadoop-mapreduce-client-common/pom.xml
@@ -19,12 +19,12 @@
   <parent>
     <artifactId>hadoop-mapreduce-client</artifactId>
     <groupId>org.apache.hadoop</groupId>
-    <version>2.6.0-cdh5.14.1-SNAPSHOT</version>
+    <version>2.6.0-cdh5.14.2</version>
   </parent>
   <modelVersion>4.0.0</modelVersion>
   <groupId>org.apache.hadoop</groupId>
   <artifactId>hadoop-mapreduce-client-common</artifactId>
-  <version>2.6.0-cdh5.14.1-SNAPSHOT</version>
+  <version>2.6.0-cdh5.14.2</version>
   <name>hadoop-mapreduce-client-common</name>
 
   <properties>
diff --git a/hadoop-mapreduce-project/hadoop-mapreduce-client/hadoop-mapreduce-client-core/pom.xml b/hadoop-mapreduce-project/hadoop-mapreduce-client/hadoop-mapreduce-client-core/pom.xml
index 5db2fa7..7b26094 100644
--- a/hadoop-mapreduce-project/hadoop-mapreduce-client/hadoop-mapreduce-client-core/pom.xml
+++ b/hadoop-mapreduce-project/hadoop-mapreduce-client/hadoop-mapreduce-client-core/pom.xml
@@ -19,12 +19,12 @@
   <parent>
     <artifactId>hadoop-mapreduce-client</artifactId>
     <groupId>org.apache.hadoop</groupId>
-    <version>2.6.0-cdh5.14.1-SNAPSHOT</version>
+    <version>2.6.0-cdh5.14.2</version>
   </parent>
   <modelVersion>4.0.0</modelVersion>
   <groupId>org.apache.hadoop</groupId>
   <artifactId>hadoop-mapreduce-client-core</artifactId>
-  <version>2.6.0-cdh5.14.1-SNAPSHOT</version>
+  <version>2.6.0-cdh5.14.2</version>
   <name>hadoop-mapreduce-client-core</name>
 
   <properties>
diff --git a/hadoop-mapreduce-project/hadoop-mapreduce-client/hadoop-mapreduce-client-hs-plugins/pom.xml b/hadoop-mapreduce-project/hadoop-mapreduce-client/hadoop-mapreduce-client-hs-plugins/pom.xml
index 56e329a..8bbe297 100644
--- a/hadoop-mapreduce-project/hadoop-mapreduce-client/hadoop-mapreduce-client-hs-plugins/pom.xml
+++ b/hadoop-mapreduce-project/hadoop-mapreduce-client/hadoop-mapreduce-client-hs-plugins/pom.xml
@@ -19,12 +19,12 @@
   <parent>
     <artifactId>hadoop-mapreduce-client</artifactId>
     <groupId>org.apache.hadoop</groupId>
-    <version>2.6.0-cdh5.14.1-SNAPSHOT</version>
+    <version>2.6.0-cdh5.14.2</version>
   </parent>
   <modelVersion>4.0.0</modelVersion>
   <groupId>org.apache.hadoop</groupId>
   <artifactId>hadoop-mapreduce-client-hs-plugins</artifactId>
-  <version>2.6.0-cdh5.14.1-SNAPSHOT</version>
+  <version>2.6.0-cdh5.14.2</version>
   <name>hadoop-mapreduce-client-hs-plugins</name>
 
   <properties>
diff --git a/hadoop-mapreduce-project/hadoop-mapreduce-client/hadoop-mapreduce-client-hs/pom.xml b/hadoop-mapreduce-project/hadoop-mapreduce-client/hadoop-mapreduce-client-hs/pom.xml
index 80b9c41..b16c6e8 100644
--- a/hadoop-mapreduce-project/hadoop-mapreduce-client/hadoop-mapreduce-client-hs/pom.xml
+++ b/hadoop-mapreduce-project/hadoop-mapreduce-client/hadoop-mapreduce-client-hs/pom.xml
@@ -19,12 +19,12 @@
   <parent>
     <artifactId>hadoop-mapreduce-client</artifactId>
     <groupId>org.apache.hadoop</groupId>
-    <version>2.6.0-cdh5.14.1-SNAPSHOT</version>
+    <version>2.6.0-cdh5.14.2</version>
   </parent>
   <modelVersion>4.0.0</modelVersion>
   <groupId>org.apache.hadoop</groupId>
   <artifactId>hadoop-mapreduce-client-hs</artifactId>
-  <version>2.6.0-cdh5.14.1-SNAPSHOT</version>
+  <version>2.6.0-cdh5.14.2</version>
   <name>hadoop-mapreduce-client-hs</name>
 
   <properties>
diff --git a/hadoop-mapreduce-project/hadoop-mapreduce-client/hadoop-mapreduce-client-jobclient/pom.xml b/hadoop-mapreduce-project/hadoop-mapreduce-client/hadoop-mapreduce-client-jobclient/pom.xml
index 24dceb4..a932e12 100644
--- a/hadoop-mapreduce-project/hadoop-mapreduce-client/hadoop-mapreduce-client-jobclient/pom.xml
+++ b/hadoop-mapreduce-project/hadoop-mapreduce-client/hadoop-mapreduce-client-jobclient/pom.xml
@@ -19,12 +19,12 @@
   <parent>
     <artifactId>hadoop-mapreduce-client</artifactId>
     <groupId>org.apache.hadoop</groupId>
-    <version>2.6.0-cdh5.14.1-SNAPSHOT</version>
+    <version>2.6.0-cdh5.14.2</version>
   </parent>
   <modelVersion>4.0.0</modelVersion>
   <groupId>org.apache.hadoop</groupId>
   <artifactId>hadoop-mapreduce-client-jobclient</artifactId>
-  <version>2.6.0-cdh5.14.1-SNAPSHOT</version>
+  <version>2.6.0-cdh5.14.2</version>
   <name>hadoop-mapreduce-client-jobclient</name>
 
   <properties>
diff --git a/hadoop-mapreduce-project/hadoop-mapreduce-client/hadoop-mapreduce-client-nativetask/pom.xml b/hadoop-mapreduce-project/hadoop-mapreduce-client/hadoop-mapreduce-client-nativetask/pom.xml
index e261513..5148d20 100644
--- a/hadoop-mapreduce-project/hadoop-mapreduce-client/hadoop-mapreduce-client-nativetask/pom.xml
+++ b/hadoop-mapreduce-project/hadoop-mapreduce-client/hadoop-mapreduce-client-nativetask/pom.xml
@@ -19,12 +19,12 @@
   <parent>
     <artifactId>hadoop-mapreduce-client</artifactId>
     <groupId>org.apache.hadoop</groupId>
-    <version>2.6.0-cdh5.14.1-SNAPSHOT</version>
+    <version>2.6.0-cdh5.14.2</version>
   </parent>
   <modelVersion>4.0.0</modelVersion>
   <groupId>org.apache.hadoop</groupId>
   <artifactId>hadoop-mapreduce-client-nativetask</artifactId>
-  <version>2.6.0-cdh5.14.1-SNAPSHOT</version>
+  <version>2.6.0-cdh5.14.2</version>
   <name>hadoop-mapreduce-client-nativetask</name>
 
   <properties>
diff --git a/hadoop-mapreduce-project/hadoop-mapreduce-client/hadoop-mapreduce-client-shuffle/pom.xml b/hadoop-mapreduce-project/hadoop-mapreduce-client/hadoop-mapreduce-client-shuffle/pom.xml
index 52f9d7d..b155ceb 100644
--- a/hadoop-mapreduce-project/hadoop-mapreduce-client/hadoop-mapreduce-client-shuffle/pom.xml
+++ b/hadoop-mapreduce-project/hadoop-mapreduce-client/hadoop-mapreduce-client-shuffle/pom.xml
@@ -19,12 +19,12 @@
   <parent>
     <artifactId>hadoop-mapreduce-client</artifactId>
     <groupId>org.apache.hadoop</groupId>
-    <version>2.6.0-cdh5.14.1-SNAPSHOT</version>
+    <version>2.6.0-cdh5.14.2</version>
   </parent>
   <modelVersion>4.0.0</modelVersion>
   <groupId>org.apache.hadoop</groupId>
   <artifactId>hadoop-mapreduce-client-shuffle</artifactId>
-  <version>2.6.0-cdh5.14.1-SNAPSHOT</version>
+  <version>2.6.0-cdh5.14.2</version>
   <name>hadoop-mapreduce-client-shuffle</name>
 
   <properties>
diff --git a/hadoop-mapreduce-project/hadoop-mapreduce-client/pom.xml b/hadoop-mapreduce-project/hadoop-mapreduce-client/pom.xml
index d6d32dc..618ade5 100644
--- a/hadoop-mapreduce-project/hadoop-mapreduce-client/pom.xml
+++ b/hadoop-mapreduce-project/hadoop-mapreduce-client/pom.xml
@@ -20,12 +20,12 @@
   <parent>
     <groupId>org.apache.hadoop</groupId>
     <artifactId>hadoop-project</artifactId>
-    <version>2.6.0-cdh5.14.1-SNAPSHOT</version>
+    <version>2.6.0-cdh5.14.2</version>
     <relativePath>../../hadoop-project</relativePath>
   </parent>
   <groupId>org.apache.hadoop</groupId>
   <artifactId>hadoop-mapreduce-client</artifactId>
-  <version>2.6.0-cdh5.14.1-SNAPSHOT</version>
+  <version>2.6.0-cdh5.14.2</version>
   <name>hadoop-mapreduce-client</name>
   <packaging>pom</packaging>
 
diff --git a/hadoop-mapreduce-project/hadoop-mapreduce-examples/pom.xml b/hadoop-mapreduce-project/hadoop-mapreduce-examples/pom.xml
index 29d7c2f..461e7ae 100644
--- a/hadoop-mapreduce-project/hadoop-mapreduce-examples/pom.xml
+++ b/hadoop-mapreduce-project/hadoop-mapreduce-examples/pom.xml
@@ -20,12 +20,12 @@
   <parent>
     <groupId>org.apache.hadoop</groupId>
     <artifactId>hadoop-project</artifactId>
-    <version>2.6.0-cdh5.14.1-SNAPSHOT</version>
+    <version>2.6.0-cdh5.14.2</version>
     <relativePath>../../hadoop-project</relativePath>
   </parent>
   <groupId>org.apache.hadoop</groupId>
   <artifactId>hadoop-mapreduce-examples</artifactId>
-  <version>2.6.0-cdh5.14.1-SNAPSHOT</version>
+  <version>2.6.0-cdh5.14.2</version>
   <description>Apache Hadoop MapReduce Examples</description>
   <name>Apache Hadoop MapReduce Examples</name>
   <packaging>jar</packaging>
diff --git a/hadoop-mapreduce-project/pom.xml b/hadoop-mapreduce-project/pom.xml
index 947b81d..08d5c0d 100644
--- a/hadoop-mapreduce-project/pom.xml
+++ b/hadoop-mapreduce-project/pom.xml
@@ -18,12 +18,12 @@
   <parent>
     <groupId>org.apache.hadoop</groupId>
     <artifactId>hadoop-project</artifactId>
-    <version>2.6.0-cdh5.14.1-SNAPSHOT</version>
+    <version>2.6.0-cdh5.14.2</version>
     <relativePath>../hadoop-project</relativePath>
   </parent>
   <groupId>org.apache.hadoop</groupId>
   <artifactId>hadoop-mapreduce</artifactId>
-  <version>2.6.0-cdh5.14.1-SNAPSHOT</version>
+  <version>2.6.0-cdh5.14.2</version>
   <packaging>pom</packaging>
   <name>hadoop-mapreduce</name>
   <url>http://hadoop.apache.org/mapreduce/</url>
diff --git a/hadoop-mapreduce1-project/build.xml b/hadoop-mapreduce1-project/build.xml
index 27dc897..95ca82f 100644
--- a/hadoop-mapreduce1-project/build.xml
+++ b/hadoop-mapreduce1-project/build.xml
@@ -54,7 +54,7 @@
  
   <property name="Name" value="Hadoop"/>
   <property name="name" value="hadoop"/>
-  <property name="version" value="2.6.0-mr1-cdh5.14.1-SNAPSHOT"/>
+  <property name="version" value="2.6.0-mr1-cdh5.14.2"/>
   <property name="final.name" value="${name}-${version}"/>
   <property name="test.final.name" value="${name}-test-${version}"/>
   <property name="year" value="2009"/>
diff --git a/hadoop-mapreduce1-project/cloudera-pom.xml b/hadoop-mapreduce1-project/cloudera-pom.xml
index 4a3b50b..67c08b2 100644
--- a/hadoop-mapreduce1-project/cloudera-pom.xml
+++ b/hadoop-mapreduce1-project/cloudera-pom.xml
@@ -26,13 +26,13 @@ NOTE - This POM is used for Cloudera's build and packaging process.
   <parent>
     <groupId>com.cloudera.cdh</groupId>
     <artifactId>hadoop-root</artifactId>
-    <version>2.6.0-mr1-cdh5.14.1-SNAPSHOT</version>
+    <version>2.6.0-mr1-cdh5.14.2</version>
     <relativePath>cloudera/maven-packaging/pom.xml</relativePath>
   </parent>
   
   <groupId>com.cloudera.cdh</groupId>
   <artifactId>hadoop-ant</artifactId>
-  <version>2.6.0-mr1-cdh5.14.1-SNAPSHOT</version>
+  <version>2.6.0-mr1-cdh5.14.2</version>
   <packaging>pom</packaging>
   
   <description>Maven wrapper for Ant Hadoop build</description>
diff --git a/hadoop-mapreduce1-project/cloudera/maven-packaging/hadoop-client/pom.xml b/hadoop-mapreduce1-project/cloudera/maven-packaging/hadoop-client/pom.xml
index 11f15cc..76570cb 100644
--- a/hadoop-mapreduce1-project/cloudera/maven-packaging/hadoop-client/pom.xml
+++ b/hadoop-mapreduce1-project/cloudera/maven-packaging/hadoop-client/pom.xml
@@ -20,12 +20,12 @@
   <parent>
     <groupId>com.cloudera.cdh</groupId>
     <artifactId>hadoop-root</artifactId>
-    <version>2.6.0-mr1-cdh5.14.1-SNAPSHOT</version>
+    <version>2.6.0-mr1-cdh5.14.2</version>
     <relativePath>../pom.xml</relativePath>
   </parent>
   <groupId>org.apache.hadoop</groupId>
   <artifactId>hadoop-client</artifactId>
-  <version>2.6.0-mr1-cdh5.14.1-SNAPSHOT</version>
+  <version>2.6.0-mr1-cdh5.14.2</version>
   <packaging>jar</packaging>
 
   <description>Apache Hadoop Client</description>
diff --git a/hadoop-mapreduce1-project/cloudera/maven-packaging/hadoop-core/pom.xml b/hadoop-mapreduce1-project/cloudera/maven-packaging/hadoop-core/pom.xml
index e0f7523..a468e2f 100644
--- a/hadoop-mapreduce1-project/cloudera/maven-packaging/hadoop-core/pom.xml
+++ b/hadoop-mapreduce1-project/cloudera/maven-packaging/hadoop-core/pom.xml
@@ -23,13 +23,13 @@
   <parent>
     <groupId>com.cloudera.cdh</groupId>
     <artifactId>hadoop-root</artifactId>
-    <version>2.6.0-mr1-cdh5.14.1-SNAPSHOT</version>
+    <version>2.6.0-mr1-cdh5.14.2</version>
     <relativePath>../pom.xml</relativePath>
   </parent>
   
   <groupId>org.apache.hadoop</groupId>
   <artifactId>hadoop-core</artifactId>
-  <version>2.6.0-mr1-cdh5.14.1-SNAPSHOT</version>
+  <version>2.6.0-mr1-cdh5.14.2</version>
   <packaging>pom</packaging>
   
   <name>hadoop-core</name>
diff --git a/hadoop-mapreduce1-project/cloudera/maven-packaging/hadoop-examples/pom.xml b/hadoop-mapreduce1-project/cloudera/maven-packaging/hadoop-examples/pom.xml
index 49d92a5..83663d5 100644
--- a/hadoop-mapreduce1-project/cloudera/maven-packaging/hadoop-examples/pom.xml
+++ b/hadoop-mapreduce1-project/cloudera/maven-packaging/hadoop-examples/pom.xml
@@ -24,13 +24,13 @@
   <parent>
     <groupId>com.cloudera.cdh</groupId>
     <artifactId>hadoop-root</artifactId>
-    <version>2.6.0-mr1-cdh5.14.1-SNAPSHOT</version>
+    <version>2.6.0-mr1-cdh5.14.2</version>
     <relativePath>../pom.xml</relativePath>
   </parent>
   
   <groupId>org.apache.hadoop</groupId>
   <artifactId>hadoop-examples</artifactId>
-  <version>2.6.0-mr1-cdh5.14.1-SNAPSHOT</version>
+  <version>2.6.0-mr1-cdh5.14.2</version>
   <packaging>pom</packaging>
   
   <name>hadoop-examples</name>
diff --git a/hadoop-mapreduce1-project/cloudera/maven-packaging/hadoop-minicluster/pom.xml b/hadoop-mapreduce1-project/cloudera/maven-packaging/hadoop-minicluster/pom.xml
index 03e3ca5..388f48d 100644
--- a/hadoop-mapreduce1-project/cloudera/maven-packaging/hadoop-minicluster/pom.xml
+++ b/hadoop-mapreduce1-project/cloudera/maven-packaging/hadoop-minicluster/pom.xml
@@ -20,12 +20,12 @@
   <parent>
     <groupId>com.cloudera.cdh</groupId>
     <artifactId>hadoop-root</artifactId>
-    <version>2.6.0-mr1-cdh5.14.1-SNAPSHOT</version>
+    <version>2.6.0-mr1-cdh5.14.2</version>
     <relativePath>../pom.xml</relativePath>
   </parent>
   <groupId>org.apache.hadoop</groupId>
   <artifactId>hadoop-minicluster</artifactId>
-  <version>2.6.0-mr1-cdh5.14.1-SNAPSHOT</version>
+  <version>2.6.0-mr1-cdh5.14.2</version>
   <packaging>jar</packaging>
 
   <description>Apache Hadoop Mini-Cluster</description>
diff --git a/hadoop-mapreduce1-project/cloudera/maven-packaging/hadoop-mrunit/pom.xml b/hadoop-mapreduce1-project/cloudera/maven-packaging/hadoop-mrunit/pom.xml
index 8f35b13..5e3a241 100644
--- a/hadoop-mapreduce1-project/cloudera/maven-packaging/hadoop-mrunit/pom.xml
+++ b/hadoop-mapreduce1-project/cloudera/maven-packaging/hadoop-mrunit/pom.xml
@@ -24,13 +24,13 @@
   <parent>
     <groupId>com.cloudera.cdh</groupId>
     <artifactId>hadoop-root</artifactId>
-    <version>2.6.0-mr1-cdh5.14.1-SNAPSHOT</version>
+    <version>2.6.0-mr1-cdh5.14.2</version>
     <relativePath>../pom.xml</relativePath>
   </parent>
   
   <groupId>org.apache.hadoop</groupId>
   <artifactId>hadoop-mrunit</artifactId>
-  <version>2.6.0-mr1-cdh5.14.1-SNAPSHOT</version>
+  <version>2.6.0-mr1-cdh5.14.2</version>
   <packaging>pom</packaging>
   
   <name>hadoop-mrunit</name>
diff --git a/hadoop-mapreduce1-project/cloudera/maven-packaging/hadoop-streaming/pom.xml b/hadoop-mapreduce1-project/cloudera/maven-packaging/hadoop-streaming/pom.xml
index 557329e..7feaf2c 100644
--- a/hadoop-mapreduce1-project/cloudera/maven-packaging/hadoop-streaming/pom.xml
+++ b/hadoop-mapreduce1-project/cloudera/maven-packaging/hadoop-streaming/pom.xml
@@ -24,13 +24,13 @@
   <parent>
     <groupId>com.cloudera.cdh</groupId>
     <artifactId>hadoop-root</artifactId>
-    <version>2.6.0-mr1-cdh5.14.1-SNAPSHOT</version>
+    <version>2.6.0-mr1-cdh5.14.2</version>
     <relativePath>../pom.xml</relativePath>
   </parent>
   
   <groupId>org.apache.hadoop</groupId>
   <artifactId>hadoop-streaming</artifactId>
-  <version>2.6.0-mr1-cdh5.14.1-SNAPSHOT</version>
+  <version>2.6.0-mr1-cdh5.14.2</version>
   <packaging>pom</packaging>
   
   <name>hadoop-streaming</name>
diff --git a/hadoop-mapreduce1-project/cloudera/maven-packaging/hadoop-tarball/pom.xml b/hadoop-mapreduce1-project/cloudera/maven-packaging/hadoop-tarball/pom.xml
index dfbe725..f0efb0e 100644
--- a/hadoop-mapreduce1-project/cloudera/maven-packaging/hadoop-tarball/pom.xml
+++ b/hadoop-mapreduce1-project/cloudera/maven-packaging/hadoop-tarball/pom.xml
@@ -24,14 +24,14 @@
   <parent>
     <groupId>com.cloudera.cdh</groupId>
     <artifactId>hadoop-root</artifactId>
-    <version>2.6.0-mr1-cdh5.14.1-SNAPSHOT</version>
+    <version>2.6.0-mr1-cdh5.14.2</version>
     <relativePath>../pom.xml</relativePath>
   </parent>
   
   <groupId>org.apache.hadoop</groupId>
   <artifactId>hadoop-tarball</artifactId>
   <packaging>pom</packaging>
-  <version>2.6.0-mr1-cdh5.14.1-SNAPSHOT</version>
+  <version>2.6.0-mr1-cdh5.14.2</version>
   
   <name>hadoop-tarball</name>
   
diff --git a/hadoop-mapreduce1-project/cloudera/maven-packaging/hadoop-test/pom.xml b/hadoop-mapreduce1-project/cloudera/maven-packaging/hadoop-test/pom.xml
index eddd65e..1f8591f 100644
--- a/hadoop-mapreduce1-project/cloudera/maven-packaging/hadoop-test/pom.xml
+++ b/hadoop-mapreduce1-project/cloudera/maven-packaging/hadoop-test/pom.xml
@@ -24,14 +24,14 @@
   <parent>
     <groupId>com.cloudera.cdh</groupId>
     <artifactId>hadoop-root</artifactId>
-    <version>2.6.0-mr1-cdh5.14.1-SNAPSHOT</version>
+    <version>2.6.0-mr1-cdh5.14.2</version>
     <relativePath>../pom.xml</relativePath>
   </parent>
   
   <groupId>org.apache.hadoop</groupId>
   <artifactId>hadoop-test</artifactId>
   <packaging>pom</packaging>
-  <version>2.6.0-mr1-cdh5.14.1-SNAPSHOT</version>
+  <version>2.6.0-mr1-cdh5.14.2</version>
   
   <name>hadoop-test</name>
   
diff --git a/hadoop-mapreduce1-project/cloudera/maven-packaging/hadoop-tools/pom.xml b/hadoop-mapreduce1-project/cloudera/maven-packaging/hadoop-tools/pom.xml
index 21bb94c..969aa29 100644
--- a/hadoop-mapreduce1-project/cloudera/maven-packaging/hadoop-tools/pom.xml
+++ b/hadoop-mapreduce1-project/cloudera/maven-packaging/hadoop-tools/pom.xml
@@ -24,14 +24,14 @@
   <parent>
     <groupId>com.cloudera.cdh</groupId>
     <artifactId>hadoop-root</artifactId>
-    <version>2.6.0-mr1-cdh5.14.1-SNAPSHOT</version>
+    <version>2.6.0-mr1-cdh5.14.2</version>
     <relativePath>../pom.xml</relativePath>
   </parent>
   
   <groupId>org.apache.hadoop</groupId>
   <artifactId>hadoop-tools</artifactId>
   <packaging>pom</packaging>
-  <version>2.6.0-mr1-cdh5.14.1-SNAPSHOT</version>
+  <version>2.6.0-mr1-cdh5.14.2</version>
   
   <name>hadoop-tools</name>
   
diff --git a/hadoop-mapreduce1-project/cloudera/maven-packaging/pom.xml b/hadoop-mapreduce1-project/cloudera/maven-packaging/pom.xml
index b0c2d02..bfa1fbd 100644
--- a/hadoop-mapreduce1-project/cloudera/maven-packaging/pom.xml
+++ b/hadoop-mapreduce1-project/cloudera/maven-packaging/pom.xml
@@ -21,13 +21,13 @@
   <parent>
     <groupId>com.cloudera.cdh</groupId>
     <artifactId>cdh-root</artifactId>
-    <version>5.14.1-SNAPSHOT</version>
+    <version>5.14.2</version>
     <relativePath>../../../../../pom.xml</relativePath>
   </parent>
   
   <groupId>com.cloudera.cdh</groupId>
   <artifactId>hadoop-root</artifactId>
-  <version>2.6.0-mr1-cdh5.14.1-SNAPSHOT</version>
+  <version>2.6.0-mr1-cdh5.14.2</version>
   <packaging>pom</packaging>
   
   <description>Hadoop root</description>
diff --git a/hadoop-mapreduce1-project/ivy/libraries.properties b/hadoop-mapreduce1-project/ivy/libraries.properties
index 43059bf..755e0ff 100644
--- a/hadoop-mapreduce1-project/ivy/libraries.properties
+++ b/hadoop-mapreduce1-project/ivy/libraries.properties
@@ -42,8 +42,8 @@ ftplet-api.version=1.0.0
 ftpserver-core.version=1.0.0
 ftpserver-deprecated.version=1.0.0-M2
 
-hadoop-common.version=2.6.0-cdh5.14.1-SNAPSHOT
-hadoop-hdfs.version=2.6.0-cdh5.14.1-SNAPSHOT
+hadoop-common.version=2.6.0-cdh5.14.2
+hadoop-hdfs.version=2.6.0-cdh5.14.2
 
 hsqldb.version=1.8.0.10
 
diff --git a/hadoop-maven-plugins/pom.xml b/hadoop-maven-plugins/pom.xml
index a7847b5..faecd12 100644
--- a/hadoop-maven-plugins/pom.xml
+++ b/hadoop-maven-plugins/pom.xml
@@ -19,7 +19,7 @@
   <parent>
     <groupId>org.apache.hadoop</groupId>
     <artifactId>hadoop-project</artifactId>
-    <version>2.6.0-cdh5.14.1-SNAPSHOT</version>
+    <version>2.6.0-cdh5.14.2</version>
     <relativePath>../hadoop-project</relativePath>
   </parent>
   <groupId>org.apache.hadoop</groupId>
diff --git a/hadoop-minicluster/pom.xml b/hadoop-minicluster/pom.xml
index b921cdf..53c491b 100644
--- a/hadoop-minicluster/pom.xml
+++ b/hadoop-minicluster/pom.xml
@@ -18,12 +18,12 @@
   <parent>
     <groupId>org.apache.hadoop</groupId>
     <artifactId>hadoop-project</artifactId>
-    <version>2.6.0-cdh5.14.1-SNAPSHOT</version>
+    <version>2.6.0-cdh5.14.2</version>
     <relativePath>../hadoop-project</relativePath>
   </parent>
   <groupId>org.apache.hadoop</groupId>
   <artifactId>hadoop-minicluster</artifactId>
-  <version>2.6.0-cdh5.14.1-SNAPSHOT</version>
+  <version>2.6.0-cdh5.14.2</version>
   <packaging>jar</packaging>
 
   <description>Apache Hadoop Mini-Cluster</description>
diff --git a/hadoop-project-dist/pom.xml b/hadoop-project-dist/pom.xml
index d3a3891..fb04d0c 100644
--- a/hadoop-project-dist/pom.xml
+++ b/hadoop-project-dist/pom.xml
@@ -20,12 +20,12 @@
   <parent>
     <groupId>org.apache.hadoop</groupId>
     <artifactId>hadoop-project</artifactId>
-    <version>2.6.0-cdh5.14.1-SNAPSHOT</version>
+    <version>2.6.0-cdh5.14.2</version>
     <relativePath>../hadoop-project</relativePath>
   </parent>
   <groupId>org.apache.hadoop</groupId>
   <artifactId>hadoop-project-dist</artifactId>
-  <version>2.6.0-cdh5.14.1-SNAPSHOT</version>
+  <version>2.6.0-cdh5.14.2</version>
   <description>Apache Hadoop Project Dist POM</description>
   <name>Apache Hadoop Project Dist POM</name>
   <packaging>pom</packaging>
diff --git a/hadoop-project/pom.xml b/hadoop-project/pom.xml
index d389689..0c66061 100644
--- a/hadoop-project/pom.xml
+++ b/hadoop-project/pom.xml
@@ -20,11 +20,11 @@
   <parent>
     <groupId>org.apache.hadoop</groupId>
     <artifactId>hadoop-main</artifactId>
-    <version>2.6.0-cdh5.14.1-SNAPSHOT</version>
+    <version>2.6.0-cdh5.14.2</version>
   </parent>
   <groupId>org.apache.hadoop</groupId>
   <artifactId>hadoop-project</artifactId>
-  <version>2.6.0-cdh5.14.1-SNAPSHOT</version>
+  <version>2.6.0-cdh5.14.2</version>
   <description>Apache Hadoop Project POM</description>
   <name>Apache Hadoop Project POM</name>
   <packaging>pom</packaging>
diff --git a/hadoop-tools/hadoop-ant/pom.xml b/hadoop-tools/hadoop-ant/pom.xml
index 39ef53c..aad8c33 100644
--- a/hadoop-tools/hadoop-ant/pom.xml
+++ b/hadoop-tools/hadoop-ant/pom.xml
@@ -20,12 +20,12 @@
   <parent>
     <groupId>org.apache.hadoop</groupId>
     <artifactId>hadoop-project</artifactId>
-    <version>2.6.0-cdh5.14.1-SNAPSHOT</version>
+    <version>2.6.0-cdh5.14.2</version>
     <relativePath>../../hadoop-project</relativePath>
   </parent>
   <groupId>org.apache.hadoop</groupId>
   <artifactId>hadoop-ant</artifactId>
-  <version>2.6.0-cdh5.14.1-SNAPSHOT</version>
+  <version>2.6.0-cdh5.14.2</version>
   <description>Apache Hadoop Ant Tasks</description>
   <name>Apache Hadoop Ant Tasks</name>
   <packaging>jar</packaging>
diff --git a/hadoop-tools/hadoop-archive-logs/pom.xml b/hadoop-tools/hadoop-archive-logs/pom.xml
index f4029a9..e0d1aa9 100644
--- a/hadoop-tools/hadoop-archive-logs/pom.xml
+++ b/hadoop-tools/hadoop-archive-logs/pom.xml
@@ -20,12 +20,12 @@
   <parent>
     <groupId>org.apache.hadoop</groupId>
     <artifactId>hadoop-project</artifactId>
-    <version>2.6.0-cdh5.14.1-SNAPSHOT</version>
+    <version>2.6.0-cdh5.14.2</version>
     <relativePath>../../hadoop-project</relativePath>
   </parent>
   <groupId>org.apache.hadoop</groupId>
   <artifactId>hadoop-archive-logs</artifactId>
-  <version>2.6.0-cdh5.14.1-SNAPSHOT</version>
+  <version>2.6.0-cdh5.14.2</version>
   <description>Apache Hadoop Archive Logs</description>
   <name>Apache Hadoop Archive Logs</name>
   <packaging>jar</packaging>
diff --git a/hadoop-tools/hadoop-archives/pom.xml b/hadoop-tools/hadoop-archives/pom.xml
index 1130e05..8bd3580 100644
--- a/hadoop-tools/hadoop-archives/pom.xml
+++ b/hadoop-tools/hadoop-archives/pom.xml
@@ -20,12 +20,12 @@
   <parent>
     <groupId>org.apache.hadoop</groupId>
     <artifactId>hadoop-project</artifactId>
-    <version>2.6.0-cdh5.14.1-SNAPSHOT</version>
+    <version>2.6.0-cdh5.14.2</version>
     <relativePath>../../hadoop-project</relativePath>
   </parent>
   <groupId>org.apache.hadoop</groupId>
   <artifactId>hadoop-archives</artifactId>
-  <version>2.6.0-cdh5.14.1-SNAPSHOT</version>
+  <version>2.6.0-cdh5.14.2</version>
   <description>Apache Hadoop Archives</description>
   <name>Apache Hadoop Archives</name>
   <packaging>jar</packaging>
diff --git a/hadoop-tools/hadoop-aws/pom.xml b/hadoop-tools/hadoop-aws/pom.xml
index d8c6646..057b0a4 100644
--- a/hadoop-tools/hadoop-aws/pom.xml
+++ b/hadoop-tools/hadoop-aws/pom.xml
@@ -19,11 +19,11 @@
   <parent>
     <groupId>org.apache.hadoop</groupId>
     <artifactId>hadoop-project</artifactId>
-    <version>2.6.0-cdh5.14.1-SNAPSHOT</version>
+    <version>2.6.0-cdh5.14.2</version>
     <relativePath>../../hadoop-project</relativePath>
   </parent>
   <artifactId>hadoop-aws</artifactId>
-  <version>2.6.0-cdh5.14.1-SNAPSHOT</version>
+  <version>2.6.0-cdh5.14.2</version>
   <name>Apache Hadoop Amazon Web Services support</name>
   <description>
     This module contains code to support integration with Amazon Web Services.
diff --git a/hadoop-tools/hadoop-azure-datalake/pom.xml b/hadoop-tools/hadoop-azure-datalake/pom.xml
index d9d4da7..46b8ca3 100644
--- a/hadoop-tools/hadoop-azure-datalake/pom.xml
+++ b/hadoop-tools/hadoop-azure-datalake/pom.xml
@@ -19,7 +19,7 @@
   <parent>
     <groupId>org.apache.hadoop</groupId>
     <artifactId>hadoop-project</artifactId>
-    <version>2.6.0-cdh5.14.1-SNAPSHOT</version>
+    <version>2.6.0-cdh5.14.2</version>
     <relativePath>../../hadoop-project</relativePath>
   </parent>
   <groupId>org.apache.hadoop</groupId>
diff --git a/hadoop-tools/hadoop-azure/pom.xml b/hadoop-tools/hadoop-azure/pom.xml
index 7d0bfde..2bcd633 100644
--- a/hadoop-tools/hadoop-azure/pom.xml
+++ b/hadoop-tools/hadoop-azure/pom.xml
@@ -19,11 +19,11 @@
   <parent>
     <groupId>org.apache.hadoop</groupId>
     <artifactId>hadoop-project</artifactId>
-    <version>2.6.0-cdh5.14.1-SNAPSHOT</version>
+    <version>2.6.0-cdh5.14.2</version>
     <relativePath>../../hadoop-project</relativePath>
   </parent>
   <artifactId>hadoop-azure</artifactId>
-  <version>2.6.0-cdh5.14.1-SNAPSHOT</version>
+  <version>2.6.0-cdh5.14.2</version>
   <name>Apache Hadoop Azure support</name>
   <description>
     This module contains code to support integration with Azure.
diff --git a/hadoop-tools/hadoop-datajoin/pom.xml b/hadoop-tools/hadoop-datajoin/pom.xml
index 53e95f0..6411b5b 100644
--- a/hadoop-tools/hadoop-datajoin/pom.xml
+++ b/hadoop-tools/hadoop-datajoin/pom.xml
@@ -17,12 +17,12 @@
   <parent>
     <groupId>org.apache.hadoop</groupId>
     <artifactId>hadoop-project</artifactId>
-    <version>2.6.0-cdh5.14.1-SNAPSHOT</version>
+    <version>2.6.0-cdh5.14.2</version>
     <relativePath>../../hadoop-project</relativePath>
   </parent>
   <groupId>org.apache.hadoop</groupId>
   <artifactId>hadoop-datajoin</artifactId>
-  <version>2.6.0-cdh5.14.1-SNAPSHOT</version>
+  <version>2.6.0-cdh5.14.2</version>
   <description>Apache Hadoop Data Join</description>
   <name>Apache Hadoop Data Join</name>
   <packaging>jar</packaging>
diff --git a/hadoop-tools/hadoop-distcp/pom.xml b/hadoop-tools/hadoop-distcp/pom.xml
index 3617688..428fac4 100644
--- a/hadoop-tools/hadoop-distcp/pom.xml
+++ b/hadoop-tools/hadoop-distcp/pom.xml
@@ -20,12 +20,12 @@
   <parent>
     <groupId>org.apache.hadoop</groupId>
     <artifactId>hadoop-project</artifactId>
-    <version>2.6.0-cdh5.14.1-SNAPSHOT</version>
+    <version>2.6.0-cdh5.14.2</version>
     <relativePath>../../hadoop-project</relativePath>
   </parent>
   <groupId>org.apache.hadoop</groupId>
   <artifactId>hadoop-distcp</artifactId>
-  <version>2.6.0-cdh5.14.1-SNAPSHOT</version>
+  <version>2.6.0-cdh5.14.2</version>
   <description>Apache Hadoop Distributed Copy</description>
   <name>Apache Hadoop Distributed Copy</name>
   <packaging>jar</packaging>
diff --git a/hadoop-tools/hadoop-extras/pom.xml b/hadoop-tools/hadoop-extras/pom.xml
index 62923b9..4d31397 100644
--- a/hadoop-tools/hadoop-extras/pom.xml
+++ b/hadoop-tools/hadoop-extras/pom.xml
@@ -20,12 +20,12 @@
   <parent>
     <groupId>org.apache.hadoop</groupId>
     <artifactId>hadoop-project</artifactId>
-    <version>2.6.0-cdh5.14.1-SNAPSHOT</version>
+    <version>2.6.0-cdh5.14.2</version>
     <relativePath>../../hadoop-project</relativePath>
   </parent>
   <groupId>org.apache.hadoop</groupId>
   <artifactId>hadoop-extras</artifactId>
-  <version>2.6.0-cdh5.14.1-SNAPSHOT</version>
+  <version>2.6.0-cdh5.14.2</version>
   <description>Apache Hadoop Extras</description>
   <name>Apache Hadoop Extras</name>
   <packaging>jar</packaging>
diff --git a/hadoop-tools/hadoop-gridmix/pom.xml b/hadoop-tools/hadoop-gridmix/pom.xml
index b66b70e..4277c1d 100644
--- a/hadoop-tools/hadoop-gridmix/pom.xml
+++ b/hadoop-tools/hadoop-gridmix/pom.xml
@@ -17,12 +17,12 @@
   <parent>
     <groupId>org.apache.hadoop</groupId>
     <artifactId>hadoop-project</artifactId>
-    <version>2.6.0-cdh5.14.1-SNAPSHOT</version>
+    <version>2.6.0-cdh5.14.2</version>
     <relativePath>../../hadoop-project</relativePath>
   </parent>
   <groupId>org.apache.hadoop</groupId>
   <artifactId>hadoop-gridmix</artifactId>
-  <version>2.6.0-cdh5.14.1-SNAPSHOT</version>
+  <version>2.6.0-cdh5.14.2</version>
   <description>Apache Hadoop Gridmix</description>
   <name>Apache Hadoop Gridmix</name>
   <packaging>jar</packaging>
diff --git a/hadoop-tools/hadoop-openstack/pom.xml b/hadoop-tools/hadoop-openstack/pom.xml
index 96fd210..67c12ab 100644
--- a/hadoop-tools/hadoop-openstack/pom.xml
+++ b/hadoop-tools/hadoop-openstack/pom.xml
@@ -19,11 +19,11 @@
   <parent>
     <groupId>org.apache.hadoop</groupId>
     <artifactId>hadoop-project</artifactId>
-    <version>2.6.0-cdh5.14.1-SNAPSHOT</version>
+    <version>2.6.0-cdh5.14.2</version>
     <relativePath>../../hadoop-project</relativePath>
   </parent>
   <artifactId>hadoop-openstack</artifactId>
-  <version>2.6.0-cdh5.14.1-SNAPSHOT</version>
+  <version>2.6.0-cdh5.14.2</version>
   <name>Apache Hadoop OpenStack support</name>
   <description>
     This module contains code to support integration with OpenStack.
diff --git a/hadoop-tools/hadoop-pipes/pom.xml b/hadoop-tools/hadoop-pipes/pom.xml
index 8115448..6103ae3 100644
--- a/hadoop-tools/hadoop-pipes/pom.xml
+++ b/hadoop-tools/hadoop-pipes/pom.xml
@@ -17,12 +17,12 @@
   <parent>
     <groupId>org.apache.hadoop</groupId>
     <artifactId>hadoop-project</artifactId>
-    <version>2.6.0-cdh5.14.1-SNAPSHOT</version>
+    <version>2.6.0-cdh5.14.2</version>
     <relativePath>../../hadoop-project</relativePath>
   </parent>
   <groupId>org.apache.hadoop</groupId>
   <artifactId>hadoop-pipes</artifactId>
-  <version>2.6.0-cdh5.14.1-SNAPSHOT</version>
+  <version>2.6.0-cdh5.14.2</version>
   <description>Apache Hadoop Pipes</description>
   <name>Apache Hadoop Pipes</name>
   <packaging>pom</packaging>
diff --git a/hadoop-tools/hadoop-rumen/pom.xml b/hadoop-tools/hadoop-rumen/pom.xml
index b5173b5..a195176 100644
--- a/hadoop-tools/hadoop-rumen/pom.xml
+++ b/hadoop-tools/hadoop-rumen/pom.xml
@@ -20,12 +20,12 @@
   <parent>
     <groupId>org.apache.hadoop</groupId>
     <artifactId>hadoop-project</artifactId>
-    <version>2.6.0-cdh5.14.1-SNAPSHOT</version>
+    <version>2.6.0-cdh5.14.2</version>
     <relativePath>../../hadoop-project</relativePath>
   </parent>
   <groupId>org.apache.hadoop</groupId>
   <artifactId>hadoop-rumen</artifactId>
-  <version>2.6.0-cdh5.14.1-SNAPSHOT</version>
+  <version>2.6.0-cdh5.14.2</version>
   <description>Apache Hadoop Rumen</description>
   <name>Apache Hadoop Rumen</name>
   <packaging>jar</packaging>
diff --git a/hadoop-tools/hadoop-sls/pom.xml b/hadoop-tools/hadoop-sls/pom.xml
index 5949ba5..acb4971 100644
--- a/hadoop-tools/hadoop-sls/pom.xml
+++ b/hadoop-tools/hadoop-sls/pom.xml
@@ -19,12 +19,12 @@
   <parent>
     <groupId>org.apache.hadoop</groupId>
     <artifactId>hadoop-project</artifactId>
-    <version>2.6.0-cdh5.14.1-SNAPSHOT</version>
+    <version>2.6.0-cdh5.14.2</version>
     <relativePath>../../hadoop-project</relativePath>
   </parent>
   <groupId>org.apache.hadoop</groupId>
   <artifactId>hadoop-sls</artifactId>
-  <version>2.6.0-cdh5.14.1-SNAPSHOT</version>
+  <version>2.6.0-cdh5.14.2</version>
   <description>Apache Hadoop Scheduler Load Simulator</description>
   <name>Apache Hadoop Scheduler Load Simulator</name>
   <packaging>jar</packaging>
diff --git a/hadoop-tools/hadoop-streaming/pom.xml b/hadoop-tools/hadoop-streaming/pom.xml
index 9acbc39..7cb1d8b 100644
--- a/hadoop-tools/hadoop-streaming/pom.xml
+++ b/hadoop-tools/hadoop-streaming/pom.xml
@@ -20,12 +20,12 @@
   <parent>
     <groupId>org.apache.hadoop</groupId>
     <artifactId>hadoop-project</artifactId>
-    <version>2.6.0-cdh5.14.1-SNAPSHOT</version>
+    <version>2.6.0-cdh5.14.2</version>
     <relativePath>../../hadoop-project</relativePath>
   </parent>
   <groupId>org.apache.hadoop</groupId>
   <artifactId>hadoop-streaming</artifactId>
-  <version>2.6.0-cdh5.14.1-SNAPSHOT</version>
+  <version>2.6.0-cdh5.14.2</version>
   <description>Apache Hadoop MapReduce Streaming</description>
   <name>Apache Hadoop MapReduce Streaming</name>
   <packaging>jar</packaging>
diff --git a/hadoop-tools/hadoop-tools-dist/pom.xml b/hadoop-tools/hadoop-tools-dist/pom.xml
index aa979ef..a339793 100644
--- a/hadoop-tools/hadoop-tools-dist/pom.xml
+++ b/hadoop-tools/hadoop-tools-dist/pom.xml
@@ -20,12 +20,12 @@
   <parent>
     <groupId>org.apache.hadoop</groupId>
     <artifactId>hadoop-project-dist</artifactId>
-    <version>2.6.0-cdh5.14.1-SNAPSHOT</version>
+    <version>2.6.0-cdh5.14.2</version>
     <relativePath>../../hadoop-project-dist</relativePath>
   </parent>
   <groupId>org.apache.hadoop</groupId>
   <artifactId>hadoop-tools-dist</artifactId>
-  <version>2.6.0-cdh5.14.1-SNAPSHOT</version>
+  <version>2.6.0-cdh5.14.2</version>
   <description>Apache Hadoop Tools Dist</description>
   <name>Apache Hadoop Tools Dist</name>
   <packaging>jar</packaging>
diff --git a/hadoop-tools/pom.xml b/hadoop-tools/pom.xml
index 2b47bf6..81640ad 100644
--- a/hadoop-tools/pom.xml
+++ b/hadoop-tools/pom.xml
@@ -20,12 +20,12 @@
   <parent>
     <groupId>org.apache.hadoop</groupId>
     <artifactId>hadoop-project</artifactId>
-    <version>2.6.0-cdh5.14.1-SNAPSHOT</version>
+    <version>2.6.0-cdh5.14.2</version>
     <relativePath>../hadoop-project</relativePath>
   </parent>
   <groupId>org.apache.hadoop</groupId>
   <artifactId>hadoop-tools</artifactId>
-  <version>2.6.0-cdh5.14.1-SNAPSHOT</version>
+  <version>2.6.0-cdh5.14.2</version>
   <description>Apache Hadoop Tools</description>
   <name>Apache Hadoop Tools</name>
   <packaging>pom</packaging>
diff --git a/hadoop-yarn-project/hadoop-yarn/hadoop-yarn-api/pom.xml b/hadoop-yarn-project/hadoop-yarn/hadoop-yarn-api/pom.xml
index af6063c..fd8aa6e 100644
--- a/hadoop-yarn-project/hadoop-yarn/hadoop-yarn-api/pom.xml
+++ b/hadoop-yarn-project/hadoop-yarn/hadoop-yarn-api/pom.xml
@@ -19,12 +19,12 @@
   <parent>
     <artifactId>hadoop-yarn</artifactId>
     <groupId>org.apache.hadoop</groupId>
-    <version>2.6.0-cdh5.14.1-SNAPSHOT</version>
+    <version>2.6.0-cdh5.14.2</version>
   </parent>
   <modelVersion>4.0.0</modelVersion>
   <groupId>org.apache.hadoop</groupId>
   <artifactId>hadoop-yarn-api</artifactId>
-  <version>2.6.0-cdh5.14.1-SNAPSHOT</version>
+  <version>2.6.0-cdh5.14.2</version>
   <name>hadoop-yarn-api</name>
 
   <properties>
diff --git a/hadoop-yarn-project/hadoop-yarn/hadoop-yarn-applications/hadoop-yarn-applications-distributedshell/pom.xml b/hadoop-yarn-project/hadoop-yarn/hadoop-yarn-applications/hadoop-yarn-applications-distributedshell/pom.xml
index fc9b42b..4d8cf41 100644
--- a/hadoop-yarn-project/hadoop-yarn/hadoop-yarn-applications/hadoop-yarn-applications-distributedshell/pom.xml
+++ b/hadoop-yarn-project/hadoop-yarn/hadoop-yarn-applications/hadoop-yarn-applications-distributedshell/pom.xml
@@ -19,12 +19,12 @@
   <parent>
     <artifactId>hadoop-yarn-applications</artifactId>
     <groupId>org.apache.hadoop</groupId>
-    <version>2.6.0-cdh5.14.1-SNAPSHOT</version>
+    <version>2.6.0-cdh5.14.2</version>
   </parent>
   <modelVersion>4.0.0</modelVersion>
   <groupId>org.apache.hadoop</groupId>
   <artifactId>hadoop-yarn-applications-distributedshell</artifactId>
-  <version>2.6.0-cdh5.14.1-SNAPSHOT</version>
+  <version>2.6.0-cdh5.14.2</version>
   <name>hadoop-yarn-applications-distributedshell</name>
 
   <properties>
diff --git a/hadoop-yarn-project/hadoop-yarn/hadoop-yarn-applications/hadoop-yarn-applications-unmanaged-am-launcher/pom.xml b/hadoop-yarn-project/hadoop-yarn/hadoop-yarn-applications/hadoop-yarn-applications-unmanaged-am-launcher/pom.xml
index 9cfc98c..649b75a 100644
--- a/hadoop-yarn-project/hadoop-yarn/hadoop-yarn-applications/hadoop-yarn-applications-unmanaged-am-launcher/pom.xml
+++ b/hadoop-yarn-project/hadoop-yarn/hadoop-yarn-applications/hadoop-yarn-applications-unmanaged-am-launcher/pom.xml
@@ -19,12 +19,12 @@
   <parent>
     <artifactId>hadoop-yarn-applications</artifactId>
     <groupId>org.apache.hadoop</groupId>
-    <version>2.6.0-cdh5.14.1-SNAPSHOT</version>
+    <version>2.6.0-cdh5.14.2</version>
   </parent>
   <modelVersion>4.0.0</modelVersion>
   <groupId>org.apache.hadoop</groupId>
   <artifactId>hadoop-yarn-applications-unmanaged-am-launcher</artifactId>
-  <version>2.6.0-cdh5.14.1-SNAPSHOT</version>
+  <version>2.6.0-cdh5.14.2</version>
   <name>hadoop-yarn-applications-unmanaged-am-launcher</name>
 
   <properties>
diff --git a/hadoop-yarn-project/hadoop-yarn/hadoop-yarn-applications/pom.xml b/hadoop-yarn-project/hadoop-yarn/hadoop-yarn-applications/pom.xml
index 6a3ab35..e0dc8bf 100644
--- a/hadoop-yarn-project/hadoop-yarn/hadoop-yarn-applications/pom.xml
+++ b/hadoop-yarn-project/hadoop-yarn/hadoop-yarn-applications/pom.xml
@@ -19,12 +19,12 @@
   <parent>
     <artifactId>hadoop-yarn</artifactId>
     <groupId>org.apache.hadoop</groupId>
-    <version>2.6.0-cdh5.14.1-SNAPSHOT</version>
+    <version>2.6.0-cdh5.14.2</version>
   </parent>
   <modelVersion>4.0.0</modelVersion>
   <groupId>org.apache.hadoop</groupId>
   <artifactId>hadoop-yarn-applications</artifactId>
-  <version>2.6.0-cdh5.14.1-SNAPSHOT</version>
+  <version>2.6.0-cdh5.14.2</version>
   <name>hadoop-yarn-applications</name>
   <packaging>pom</packaging>
 
diff --git a/hadoop-yarn-project/hadoop-yarn/hadoop-yarn-client/pom.xml b/hadoop-yarn-project/hadoop-yarn/hadoop-yarn-client/pom.xml
index 2c21f7a..0e0f957 100644
--- a/hadoop-yarn-project/hadoop-yarn/hadoop-yarn-client/pom.xml
+++ b/hadoop-yarn-project/hadoop-yarn/hadoop-yarn-client/pom.xml
@@ -17,11 +17,11 @@
   <parent>
     <artifactId>hadoop-yarn</artifactId>
     <groupId>org.apache.hadoop</groupId>
-    <version>2.6.0-cdh5.14.1-SNAPSHOT</version>
+    <version>2.6.0-cdh5.14.2</version>
   </parent>
   <groupId>org.apache.hadoop</groupId>
   <artifactId>hadoop-yarn-client</artifactId>
-  <version>2.6.0-cdh5.14.1-SNAPSHOT</version>
+  <version>2.6.0-cdh5.14.2</version>
   <name>hadoop-yarn-client</name>
 
   <properties>
diff --git a/hadoop-yarn-project/hadoop-yarn/hadoop-yarn-common/pom.xml b/hadoop-yarn-project/hadoop-yarn/hadoop-yarn-common/pom.xml
index 86222f4..ac16be1 100644
--- a/hadoop-yarn-project/hadoop-yarn/hadoop-yarn-common/pom.xml
+++ b/hadoop-yarn-project/hadoop-yarn/hadoop-yarn-common/pom.xml
@@ -19,12 +19,12 @@
   <parent>
     <artifactId>hadoop-yarn</artifactId>
     <groupId>org.apache.hadoop</groupId>
-    <version>2.6.0-cdh5.14.1-SNAPSHOT</version>
+    <version>2.6.0-cdh5.14.2</version>
   </parent>
   <modelVersion>4.0.0</modelVersion>
   <groupId>org.apache.hadoop</groupId>
   <artifactId>hadoop-yarn-common</artifactId>
-  <version>2.6.0-cdh5.14.1-SNAPSHOT</version>
+  <version>2.6.0-cdh5.14.2</version>
   <name>hadoop-yarn-common</name>
 
   <properties>
diff --git a/hadoop-yarn-project/hadoop-yarn/hadoop-yarn-registry/pom.xml b/hadoop-yarn-project/hadoop-yarn/hadoop-yarn-registry/pom.xml
index e168d2c..b522d01 100644
--- a/hadoop-yarn-project/hadoop-yarn/hadoop-yarn-registry/pom.xml
+++ b/hadoop-yarn-project/hadoop-yarn/hadoop-yarn-registry/pom.xml
@@ -19,11 +19,11 @@
   <parent>
     <artifactId>hadoop-yarn</artifactId>
     <groupId>org.apache.hadoop</groupId>
-    <version>2.6.0-cdh5.14.1-SNAPSHOT</version>
+    <version>2.6.0-cdh5.14.2</version>
   </parent>
   <modelVersion>4.0.0</modelVersion>
   <artifactId>hadoop-yarn-registry</artifactId>
-  <version>2.6.0-cdh5.14.1-SNAPSHOT</version>
+  <version>2.6.0-cdh5.14.2</version>
   <name>hadoop-yarn-registry</name>
 
   <properties>
diff --git a/hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-applicationhistoryservice/pom.xml b/hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-applicationhistoryservice/pom.xml
index 246573c..5cf0199 100644
--- a/hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-applicationhistoryservice/pom.xml
+++ b/hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-applicationhistoryservice/pom.xml
@@ -22,12 +22,12 @@
   <parent>
     <artifactId>hadoop-yarn-server</artifactId>
     <groupId>org.apache.hadoop</groupId>
-    <version>2.6.0-cdh5.14.1-SNAPSHOT</version>
+    <version>2.6.0-cdh5.14.2</version>
   </parent>
   <modelVersion>4.0.0</modelVersion>
   <groupId>org.apache.hadoop</groupId>
   <artifactId>hadoop-yarn-server-applicationhistoryservice</artifactId>
-  <version>2.6.0-cdh5.14.1-SNAPSHOT</version>
+  <version>2.6.0-cdh5.14.2</version>
   <name>hadoop-yarn-server-applicationhistoryservice</name>
 
   <properties>
diff --git a/hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-common/pom.xml b/hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-common/pom.xml
index 2eed15f..35a6250 100644
--- a/hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-common/pom.xml
+++ b/hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-common/pom.xml
@@ -19,12 +19,12 @@
   <parent>
     <artifactId>hadoop-yarn-server</artifactId>
     <groupId>org.apache.hadoop</groupId>
-    <version>2.6.0-cdh5.14.1-SNAPSHOT</version>
+    <version>2.6.0-cdh5.14.2</version>
   </parent>
   <modelVersion>4.0.0</modelVersion>
   <groupId>org.apache.hadoop</groupId>
   <artifactId>hadoop-yarn-server-common</artifactId>
-  <version>2.6.0-cdh5.14.1-SNAPSHOT</version>
+  <version>2.6.0-cdh5.14.2</version>
   <name>hadoop-yarn-server-common</name>
 
   <properties>
diff --git a/hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-nodemanager/pom.xml b/hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-nodemanager/pom.xml
index c712054..242722b 100644
--- a/hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-nodemanager/pom.xml
+++ b/hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-nodemanager/pom.xml
@@ -19,12 +19,12 @@
   <parent>
     <artifactId>hadoop-yarn-server</artifactId>
     <groupId>org.apache.hadoop</groupId>
-    <version>2.6.0-cdh5.14.1-SNAPSHOT</version>
+    <version>2.6.0-cdh5.14.2</version>
   </parent>
   <modelVersion>4.0.0</modelVersion>
   <groupId>org.apache.hadoop</groupId>
   <artifactId>hadoop-yarn-server-nodemanager</artifactId>
-  <version>2.6.0-cdh5.14.1-SNAPSHOT</version>
+  <version>2.6.0-cdh5.14.2</version>
   <name>hadoop-yarn-server-nodemanager</name>
 
   <properties>
diff --git a/hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-resourcemanager/pom.xml b/hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-resourcemanager/pom.xml
index bc65087..ea9189c 100644
--- a/hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-resourcemanager/pom.xml
+++ b/hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-resourcemanager/pom.xml
@@ -19,12 +19,12 @@
   <parent>
     <artifactId>hadoop-yarn-server</artifactId>
     <groupId>org.apache.hadoop</groupId>
-    <version>2.6.0-cdh5.14.1-SNAPSHOT</version>
+    <version>2.6.0-cdh5.14.2</version>
   </parent>
   <modelVersion>4.0.0</modelVersion>
   <groupId>org.apache.hadoop</groupId>
   <artifactId>hadoop-yarn-server-resourcemanager</artifactId>
-  <version>2.6.0-cdh5.14.1-SNAPSHOT</version>
+  <version>2.6.0-cdh5.14.2</version>
   <name>hadoop-yarn-server-resourcemanager</name>
 
   <properties>
diff --git a/hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-tests/pom.xml b/hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-tests/pom.xml
index 9d8ffb3..cc39a37 100644
--- a/hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-tests/pom.xml
+++ b/hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-tests/pom.xml
@@ -19,11 +19,11 @@
   <parent>
     <artifactId>hadoop-yarn-server</artifactId>
     <groupId>org.apache.hadoop</groupId>
-    <version>2.6.0-cdh5.14.1-SNAPSHOT</version>
+    <version>2.6.0-cdh5.14.2</version>
   </parent>
   <groupId>org.apache.hadoop</groupId>
   <artifactId>hadoop-yarn-server-tests</artifactId>
-  <version>2.6.0-cdh5.14.1-SNAPSHOT</version>
+  <version>2.6.0-cdh5.14.2</version>
   <name>hadoop-yarn-server-tests</name>
 
   <properties>
diff --git a/hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-web-proxy/pom.xml b/hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-web-proxy/pom.xml
index 9361f54..9310fcd 100644
--- a/hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-web-proxy/pom.xml
+++ b/hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-web-proxy/pom.xml
@@ -19,12 +19,12 @@
   <parent>
     <artifactId>hadoop-yarn-server</artifactId>
     <groupId>org.apache.hadoop</groupId>
-    <version>2.6.0-cdh5.14.1-SNAPSHOT</version>
+    <version>2.6.0-cdh5.14.2</version>
   </parent>
   <modelVersion>4.0.0</modelVersion>
   <groupId>org.apache.hadoop</groupId>
   <artifactId>hadoop-yarn-server-web-proxy</artifactId>
-  <version>2.6.0-cdh5.14.1-SNAPSHOT</version>
+  <version>2.6.0-cdh5.14.2</version>
   <name>hadoop-yarn-server-web-proxy</name>
 
   <properties>
diff --git a/hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/pom.xml b/hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/pom.xml
index 245d145..1df8e88 100644
--- a/hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/pom.xml
+++ b/hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/pom.xml
@@ -19,12 +19,12 @@
   <parent>
     <artifactId>hadoop-yarn</artifactId>
     <groupId>org.apache.hadoop</groupId>
-    <version>2.6.0-cdh5.14.1-SNAPSHOT</version>
+    <version>2.6.0-cdh5.14.2</version>
   </parent>
   <modelVersion>4.0.0</modelVersion>
   <groupId>org.apache.hadoop</groupId>
   <artifactId>hadoop-yarn-server</artifactId>
-  <version>2.6.0-cdh5.14.1-SNAPSHOT</version>
+  <version>2.6.0-cdh5.14.2</version>
   <name>hadoop-yarn-server</name>
   <packaging>pom</packaging>
 
diff --git a/hadoop-yarn-project/hadoop-yarn/hadoop-yarn-site/pom.xml b/hadoop-yarn-project/hadoop-yarn/hadoop-yarn-site/pom.xml
index 8a73479..38ff2cd 100644
--- a/hadoop-yarn-project/hadoop-yarn/hadoop-yarn-site/pom.xml
+++ b/hadoop-yarn-project/hadoop-yarn/hadoop-yarn-site/pom.xml
@@ -19,12 +19,12 @@
   <parent>
     <artifactId>hadoop-yarn</artifactId>
     <groupId>org.apache.hadoop</groupId>
-    <version>2.6.0-cdh5.14.1-SNAPSHOT</version>
+    <version>2.6.0-cdh5.14.2</version>
   </parent>
   <modelVersion>4.0.0</modelVersion>
   <groupId>org.apache.hadoop</groupId>
   <artifactId>hadoop-yarn-site</artifactId>
-  <version>2.6.0-cdh5.14.1-SNAPSHOT</version>
+  <version>2.6.0-cdh5.14.2</version>
   <name>hadoop-yarn-site</name>
   <packaging>pom</packaging>
 
diff --git a/hadoop-yarn-project/hadoop-yarn/pom.xml b/hadoop-yarn-project/hadoop-yarn/pom.xml
index b19946e..fdae5b6 100644
--- a/hadoop-yarn-project/hadoop-yarn/pom.xml
+++ b/hadoop-yarn-project/hadoop-yarn/pom.xml
@@ -16,12 +16,12 @@
   <parent>
     <groupId>org.apache.hadoop</groupId>
     <artifactId>hadoop-project</artifactId>
-    <version>2.6.0-cdh5.14.1-SNAPSHOT</version>
+    <version>2.6.0-cdh5.14.2</version>
     <relativePath>../../hadoop-project</relativePath>
   </parent>
   <groupId>org.apache.hadoop</groupId>
   <artifactId>hadoop-yarn</artifactId>
-  <version>2.6.0-cdh5.14.1-SNAPSHOT</version>
+  <version>2.6.0-cdh5.14.2</version>
   <packaging>pom</packaging>
   <name>hadoop-yarn</name>
 
diff --git a/hadoop-yarn-project/pom.xml b/hadoop-yarn-project/pom.xml
index 2582584..84b5f4e 100644
--- a/hadoop-yarn-project/pom.xml
+++ b/hadoop-yarn-project/pom.xml
@@ -18,12 +18,12 @@
   <parent>
     <groupId>org.apache.hadoop</groupId>
     <artifactId>hadoop-project</artifactId>
-    <version>2.6.0-cdh5.14.1-SNAPSHOT</version>
+    <version>2.6.0-cdh5.14.2</version>
     <relativePath>../hadoop-project</relativePath>
   </parent>
   <groupId>org.apache.hadoop</groupId>
   <artifactId>hadoop-yarn-project</artifactId>
-  <version>2.6.0-cdh5.14.1-SNAPSHOT</version>
+  <version>2.6.0-cdh5.14.2</version>
   <packaging>pom</packaging>
   <name>hadoop-yarn-project</name>
   <url>http://hadoop.apache.org/yarn/</url>
diff --git a/pom.xml b/pom.xml
index e59392e..c51acbc 100644
--- a/pom.xml
+++ b/pom.xml
@@ -19,11 +19,11 @@ xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xs
   <parent>
     <groupId>com.cloudera.cdh</groupId>
     <artifactId>cdh-root</artifactId>
-    <version>5.14.1-SNAPSHOT</version>
+    <version>5.14.2</version>
   </parent>
   <groupId>org.apache.hadoop</groupId>
   <artifactId>hadoop-main</artifactId>
-  <version>2.6.0-cdh5.14.1-SNAPSHOT</version>
+  <version>2.6.0-cdh5.14.2</version>
   <description>Apache Hadoop Main</description>
   <name>Apache Hadoop Main</name>
   <packaging>pom</packaging>
-- 
1.7.9.5

